Source/Cake.DotNetTool.Module/DotNetToolContentResolver.cs

```diff
@@ -69,6 +69,10 @@ public IReadOnlyCollection<IFile> GetFiles(PackageReference package, PackageType
                         return GetToolFiles(new DirectoryPath(_environment.GetEnvironmentVariable("USERPROFILE")).Combine(".dotnet/tools"), package);
                     }
                 }
+                else if (package.Parameters.ContainsKey("local"))
+                {
+                    return new List<IFile>(); //TODO

```

Comment:
Not sure what, if anything, can be done here, since dotnet local tools are not put in the user subdirectory like dotnet global tools.

Only idea is they must be in the NuGet cache, but how to get the correct path?

Failing that, would need to revisit how tools work and allow them to be managed behind the dotnet CLI.